indigo_app/js/components/TaxonomyTOC.vue

```diff
@@ -27,15 +27,15 @@ export default {
       const anchor = tocItem.querySelector('.content__action__title');
       const href = new URLSearchParams(anchor.getAttribute('href'));
       if (params.get('taxonomy_topic') === href.get('taxonomy_topic')) {
-        anchor.classList.add('active');
+        anchor.classList.add('active-taxonomy');
       }
     });
   }
 };
 </script>
 
 <style>
```

Comment:
```suggestion
<style scoped>
```

Reply:
that'll scope this only to this component.

alternatively scope the CSS selector to the sidebar and/or `la-table-of-contents-controller`

Reply:
@actlikewill my preferred mechanism would be to add a wrapping `taxonomy-sidebar` class to the whole sidebar, and then move this CSS into the app's actual SCSS files (rather than here), and then scope the `.active` styles to just the sidebar.

Reply:
@longhotsummer This is not working. I think its because the style added dynamically with JS

Reply:
@longhotsummer I went with the second suggestion. Thanks